balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/CassandraUtil.scala

```diff
@@ -41,6 +41,12 @@ object CassandraUtil extends StrictLogging {
       session.close()
       result
     }
+
+    def executeUpdate(stmt: Statement): Unit = {
+      val session = this.newSession
+      session.execute(stmt)
+      session.close()
```

Comment:
The `.close()` should be in the finally clause of a try block, so if there are any exceptions, it gets closed in any case. Same with the method above. We overlooked the error case last time.